packages/components/Button/src/FAB/FABColorTokens.ts

```diff
@@ -1,15 +1,14 @@
 import { Theme } from '@fluentui-react-native/framework';
 import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { ButtonCoreTokens } from '../Button.types';
-import { shadowStyleFromTheme } from '../shadowStyle';
 
 export const defaultFABColorTokens: TokenSettings<ButtonCoreTokens, Theme> = (t: Theme): ButtonCoreTokens => ({
-  ...shadowStyleFromTheme(t, 'shadow8'),
   // coloring same as primary
   backgroundColor: t.colors.brandedBackground,
   color: t.colors.brandedContent,
   borderColor: t.colors.brandedBorder,
   iconColor: t.colors.brandedIcon,
+  shadowDepth: 'shadow8',
```

Comment:
Since this isn't a color token, maybe move this to `FABTokens.ts`?
Also, `shadowDepth: t.shadows.shadow8`?

Reply:
For moving it to FABTokens.ts - this is where the shadow tokens from before were initially defined, and also when I briefly did macOS I put the tokens in ButtonColorTokens.macos.ts since I can specify which variants have shadows there. I do think it makes more sense in FABTokens.ts but wasn't sure if there was reasoning behind why the shadows were initially defined there

Reply:
I was probably just lazy and put it there is the answer. Or I told someone else to stick it in there.

Reply:
Planning to refactor stuff so I can pass in t.shadows.shadow8 in a later PR